specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/network.json

```diff
@@ -0,0 +1,439 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2022-06-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
```

Comment:
There are a bunch of errors in the open-api-pipeline. Please check and address them.

[https://github.com/Azure/azure-rest-api-specs/pull/21458/checks?check_run_id=9351184173](https://github.com/Azure/azure-rest-api-specs/pull/21458/checks?check_run_id=9351184173)